pkg/sql/schema_changer_test.go

```diff
@@ -1356,6 +1364,15 @@ func TestSchemaChangeRetryOnVersionChange(t *testing.T) {
 				if currChunk == 3 {
 					// Publish a new version of the table.
 					upTableVersion()
+
+					// TODO(adityamaru): Previously, the index backfiller would
```

Comment:
Is not observing changes in table desc version throughout an index backfill okay? This was not the case in the old implementation as we would re-acquire the table desc lease after every 2mins/80% buffer fill as part of the DistSQL flow planning. This in turn would throw a version mismatch error and our schema change would be retried.

Reply:
I'll defer to @lucy-zhang or @ajwerner on this one.